recipes/default.rb

```diff
@@ -48,17 +48,16 @@
         end
       end
     end
-  end
 
-  service "networking" do
-    supports :restart => true
-    action :nothing
-  end
 
-  node['ucarp']['bonded_interfaces'].each do |interface|
-    execute "flush ip on #{interface}" do
-      command "ip addr flush dev #{interface}"
-      action :nothing
-    end
+service 'networking' do
+  supports restart: true
+  action :nothing
+end
+
+node['ucarp']['bonded_interfaces'].each do |interface|
+  execute "flush ip on #{interface}" do
+    command "ip addr flush dev #{interface}"
+    action :nothing
   end
 end
```

Comment:
The service and node['ucarp'] blocks need to be under the unless block. So these need to be indented with an additional `end` on the bottom.